From 12cf0ff7d80ac8397c44192a6e625f9243560c32 Mon Sep 17 00:00:00 2001
From: Kosta Zertsekel <zertsekel@gmail.com>
Date: Fri, 18 Jan 2013 18:06:06 +0200
Subject: [PATCH 0450/1825] ALP: Remove unused mach/vmalloc.h

https://github.com/MISL-EBU-System-SW/misl-windriver.git linux-3.4.69-14t2-read
commit 1897023ccd023f7ab2aeed5641744d67d142d40f

Since mach/vmalloc.h is not used in linux 3.4 anymore.

Signed-off-by: Kosta Zertsekel <konszert@marvell.com>
Change-Id: I1a4a230c54cd7371b77b0e13847633c1bec71d86
Reviewed-on: http://vgitil04.il.marvell.com:8080/1226
Reviewed-by: Eran Ben-Avi <benavi@marvell.com>
Tested-by: Eran Ben-Avi <benavi@marvell.com>
Signed-off-by: Zhong Hongbo <hongbo.zhong@windriver.com>
---
 arch/arm/mach-avantalp/include/mach/vmalloc.h |   11 -----------
 1 files changed, 0 insertions(+), 11 deletions(-)
 delete mode 100644 arch/arm/mach-avantalp/include/mach/vmalloc.h

diff --git a/arch/arm/mach-avantalp/include/mach/vmalloc.h b/arch/arm/mach-avantalp/include/mach/vmalloc.h
deleted file mode 100644
index 1630fe6..0000000
--- a/arch/arm/mach-avantalp/include/mach/vmalloc.h
+++ /dev/null
@@ -1,11 +0,0 @@
-/*
- * include/asm-arm/arch-aurora/vmalloc.h
- */
-
-/* Dove LCD driver performs big allocations for FrameBuffer memory, we need to
- * move CONSISTENT_BASE by 32MB
- */
-/* Was 0x2000000 */
-
-#define VMALLOC_END	(0xfa800000)
-
-- 
1.7.5.4

